app/common/locales/en.json

```diff
@@ -198,6 +198,8 @@
         "add_task" : "Add Task",
         "title" : "Title",
         "field_allowed_relation_survey": "Allowed surveys",
+        "disable_caption_field": "Disable caption field",
```

Comment:
I'd reverse this, similar to the actual config value ie. `Show image caption` or `Collect image caption`

Reply:
Ah. OK yes, that makes sense.

Reply:
mmmm... I'm asking Jess about this. I'm not sure I like the explanation bit when it's reversed (and I don't know what to change it for)